src/Smalot/PdfParser/Page.php

```diff
@@ -673,20 +673,28 @@ public function getDataTm(array $dataCommands = null): array
          *  Set default values for font data
          */
         $defaultFontId = -1;
-        $defaultFontSize = 0;
+        $defaultFontSize = 1;
 
         /*
-         * Setting where are the X and Y coordinates in the matrix (Tm)
+         * Setting where the scaling and X,Y-coordinates in the matrix are (Tm)
          */
+        $hSc = 0; // horizontal scaling
+        $vSc = 3; // vertical scaling
```

Comment:
Out of curiosity, why using `3` as default value for vertical scaling? Is there a reference in the PDF spec for instance?

Reply:
@k00ni It is not the value of the vertical scaling, but the index of vertical scaling the  in the array that encodes the text matrix.

![grafik](https://user-images.githubusercontent.com/45591639/208697176-b6445178-d7f7-438e-8ed1-5b088e8308ce.png)

So it corresponds to `d` here.